andela/travel_tool_front
# #160710678 Enable travel administrators create travel team members

#### What does this PR do?
Allows travel administrators to create travel team members on the Travel application

#### Description of Task to be completed?
- Create actions, sagas and reducers for fetching Andela centers
- Modify existing actions, sagas and reducers to fetch travel team members
- Modify existing actions, sagas and reducers to create travel team members
- Create component and view for viewing travel team members
- Write tests for components, actions, reducers and sagas

#### How should this be manually tested?
- Clone the repository
- Run `make start` docker build command on the terminal to install all dependencies and run the App in the docker development environment server.
- On the browser, type the localhost URL ` http://localhost:3000` and Login with your Andela email account into the App.
- Change your role in the UserRole table to travel admin
- On the left side bar, click on the `Travel team` tab 
- You should see the travel team members displayed
- Clicking on `add user` to add a new travel team member

#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?
[#160710678](https://www.pivotaltracker.com/story/show/160710678)

#### Screenshots (if appropriate)
<img width="1440" alt="screenshot 2018-10-18 at 5 17 03 pm" src="https://user-images.githubusercontent.com/25195857/47168865-f130d880-d2f9-11e8-8297-6f5bb8cc57ca.png">

#### Questions:
N/A
